packages/react-table/src/components/Table/utils/decorators/treeRow.tsx

```diff
@@ -61,6 +66,21 @@ export const treeRow = (onCollapse: OnTreeRowCollapse, onCheckChange?: OnCheckCh
             </span>
           )}
           {text}
+          {!!onToggleRowDetails && (
+            <span className={css(stylesTreeView.tableTreeViewDetailsToggle)}>
+              <Button
+                variant="plain"
+                aria-expanded={isDetailsExpanded}
+                className={css(isDetailsExpanded && styles.modifiers.expanded)}
```

Comment:
You shouldn't need this. It isn't in the core docs and I don't see any CSS for it.

```suggestion
```